lib/rubocop/cop/style/hash_syntax.rb

```diff
@@ -96,7 +96,7 @@ def word_symbol_pair?(pair)
           if key.sym_type?
             sym_name = key.loc.expression.source
 
-            sym_name !~ /\A:["']/
+            sym_name !~ /\A:["']/ && !sym_name.end_with?('=')
```

Comment:
Since there is already a regex match happening, would it make more sense to write the line as
```ruby
sym_name !~ /\A:["']|[^=]\z/
```

I think that the code is easier to understand as it is currently written, but it is something that popped into my mind so I figured that I would mention it.